test/unit/specs/events/cd-event-add-child-tickets.spec.js

```diff
@@ -0,0 +1,314 @@
+            { approvedApplications: 0,
+              deleted: 0,
+              entity: 'ticketentity',
+              id: 'ticketId2',
+              invites: null,
+              name: 'Ticket2',
+              quantity: 1,
+              sessionId: 'sessionId',
+              totalApplications: 0,
+              type: 'ninja' }],
+        }];
+
+        // ACT
+
+        // ASSERT
+        expect(vm.childTickets).to.deep.equal([{ description: 'description1',
+          entity: 'sessionentity',
+          eventId: 'some-eventId',
+          id: 'sessionId',
+          name: 'Session1',
+          status: 'status1',
+          tickets: [
+            { approvedApplications: 0,
+              deleted: 0,
+              entity: 'ticketentity',
+              id: 'ticketId2',
+              invites: null,
+              name: 'Ticket2',
+              quantity: 1,
+              sessionId: 'sessionId',
+              totalApplications: 0,
+              type: 'ninja' }],
+        }]);
+      });
+    });
+
+    describe('computed.name', () => {
+      it('should return the full name - firstName and surname contactenated', async () => {
+        // ARRANGE
+        const vm = vueUnitHelper(ChildTicketWithMocks);
+        vm.firstName = 'Jane';
+        vm.surname = 'Doe';
+
+        // ACT
+
+        // ASSERT
+        expect(vm.name).to.equal('Jane Doe');
+      });
+    });
+
+    describe('computed.status', () => {
+      it('should return a status of pending if the event ticketApproval is set to true', async () => {
+        // ARRANGE
+        const vm = vueUnitHelper(ChildTicketWithMocks);
+        vm.event = { ticketApproval: true };
+
+        // ACT
+
+        // ASSERT
+        expect(vm.status).to.equal('pending');
+      });
+
+      it('should return a status of approved if the event ticketApproval is set to false', async () => {
+        // ARRANGE
+        const vm = vueUnitHelper(ChildTicketWithMocks);
+        vm.event = { ticketApproval: false };
+
+        // ACT
+
+        // ASSERT
+        expect(vm.status).to.equal('approved');
+      });
+    });
+
+    describe('computed.applications', () => {
+      it('should return full application when form is filled out', async () => {
+        // ARRANGE
+        const vm = vueUnitHelper(ChildTicketWithMocks);
+        vm.name = 'Jane Doe';
+        vm.dob = new Date(1980, 10, 25, 0, 0, 0, 0);
+        vm.eventId = 'eventId';
+        vm.status = 'pending';
+        vm.userId = null;
+        vm.tickets = [{
+          approvedApplications: 0,
+          deleted: 0,
+          entity: 'ticketentity',
+          id: 'ticketId2',
+          invites: null,
+          name: 'Ticket2',
+          quantity: 1,
+          sessionId: 'sessionId',
+          totalApplications: 0,
+          type: 'ninja' }];
+        vm.event = { dojoId: 'dojoId' };
+
+        // ACT
+
+        // ASSERT
+        expect(vm.applications).to.deep.equal([{
+          created: new Date(),
```

Comment:
This is very likely what is failing your tests unless you use something like http://sinonjs.org/releases/v1.17.7/fake-timers/